.travis.yml

```diff
@@ -11,8 +11,8 @@ script:
   - est --dir=est --file=/dev/null
   - mvn clean install -Pqulice -Pjacoco --errors --batch-mode
 before_install:
-  - rvm install 2.2.2
-  - rvm use 2.2.2
+  - rvm install 2.6.0
```

Comment:
@zCRUSADERz  Why did you bump the version? This was not part of the task.

Reply:
@Serranya I needed a successful build in Travis CI. Probably I should wait until #981 PR is accepted?

Reply:
@zCRUSADERz exactly. Please remove this part of the commit and wait for #981

Reply:
@Serranya deleted.